mart/attack/adversary.py

```diff
@@ -101,8 +105,8 @@ def __init__(
             assert self._attacker.max_epochs == 0
             assert self._attacker.limit_train_batches > 0
 
-        # TODO: Make this configurable. E.g. [0,1] <-> [0,255]
-        self.transform = self.untransform = lambda x: x
+        self.transform = transform if transform is not None else lambda x: x
+        self.untransform = untransform if untransform is not None else lambda x: x
```

Comment:
Why is there no use of this in this PR?

Reply:
Oh I see. Yeah, please target this at `main`. Hard to review otherwise.

Reply:
Sorry about that. I have made all related changes visible here.